pcmdi_metrics/precip_variability/scripts_pcmdi/calc_ps_area_freq_mean_regional.py

```diff
@@ -0,0 +1,152 @@
+import copy
+import datetime
+import json
+import os
+import pickle
+
+import numpy as np
+
+
+def prdday_to_frq3hridx(prdday, frequency):
```

Comment:
If you want to shorten code [L74-130], the functions `prdday_to_frq3hridx` and `prdday_to_frq1didx` can be combined into one as below.

https://github.com/PCMDI/pcmdi_metrics/blob/035f654da98301538c8a1c693449435b346c37fb/pcmdi_metrics/precip_variability/lib/lib_variability_across_timescales.py#L466-L478

Reply:
Thanks! I've copied this function into these scripts.